### ISSUE TITLE
EVSS PCIU Migration: Step 6b: Remove PCIU configuration and references from Vets-API (non-prefill)

### ISSUE THREAD

**Issue by RachalCassity** [department-of-veterans-affairs/va.gov-team#76175]
## User Story
As a developer, I want to remove all references of PCIU to ensure Vets-API does not use PCIU, as we transition to VA Profile in preparation for the PCIU deprecation

## Issue Description

PCIU configuration and references in Vets-API are no longer needed and should be removed to clean up the codebase and reduce unused configuration. This includes all references that are NOT related to prefill, including configurations and specs

`remove_pciu` flipper still allows PCIU data to fill in when data is missing. `remove_pciu_2` does not allow PCIU data to fill in when data is missing. Any reference to both `remove_pciu` and `remove_pciu_2` flippers will need to be removed with the following tasks:

## Tasks
- [x] Remove lib/evss/pciu
- [x] Remove lib/evss/pciu_address
- [x] Remove from rubocop
- [x] Remove from breakers
- [x] Remove from FormProfile
- [x] Remove from User model
- [x] Remove related spec files
- [x] Remove redis configuration
- [x] Remove settings config
- [x] Tests are passing
- [ ] ~Test locally~
- [ ] ~Test in staging~

## Acceptance Criteria
- [ ] ~PCIU configurations listed under the tasks have been completely removed from Vets-API~
- [x] PR is created and is ready for review after all blockers are removed.

**Comment by AshleyGuerrant** [comment]
Moving out 2 sprints. This work should happen after ContactInformation is complete. ContactInformation is replacing PCIU, these tickets cannot be worked on until ContactInformation has been done and the PCIU has been replaced.

**Comment by RachalCassity** [comment]
Contact Information V2 has to replace PCIU in prod before PCIU references can be removed.

**Comment by AshleyGuerrant** [comment]
**Standup note:** Moving this ticket to blocked status, as it is intricately related to and blocked by [Step 6a](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76166). Also, Step 6a has a dependency with the [mobile migration and testing](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97889) and a draft PR can be created but it cannot be merged until the mobile work is complete.
Work on this ticket can start either after the draft PR is created for Step 6a, or after the mobile migration and testing is merged.

**Comment by AshleyGuerrant** [comment]
**Sprint reporting note:** this ticket is blocked until the mobile testing is complete. Moving out to Sprint 20, but I will comment & unblock this ticket if the mobile testing happens to get completed during Sprint 19.

**Comment by RachalCassity** [comment]
Mobile pull request was merged yesterday, 2/10/2025. This work is no longer blocked and should be added to the next sprint.

**Comment by AshleyGuerrant** [comment]
**Sprint reporting note:** This ticket will roll over to Sprint 22, because it was blocked until today (2/25/25). Mobile [testing was complete](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97889#issuecomment-2654702670) as of 2/12/25. However, the VFS teams testing was just fully completed today, now that #98925 is done.

**Comment by AshleyGuerrant** [comment]
@RachalCassity - any updates on this?

**Comment by AshleyGuerrant** [comment]
**Sprint Planning note:** @RachalCassity is going to write up a separate ticket to track the PR merge for this work. The majority of this work is already complete, but we are blocked on merging the PR until the[ VAProfile API key is obtained](https://github.com/department-of-veterans-affairs/va.gov-team/issues/103958).
